javascript-functions/main.js

```diff
@@ -0,0 +1,35 @@
+function convertMinutesToSeconds(minutes, seconds) {
+  var multiply = minutes * seconds;
+  return multiply;
+}
+
+var convertMinutesToSecondsResult = convertMinutesToSeconds(5, 60);
+console.log('convertMinutesToSecondsResult:', convertMinutesToSecondsResult);
+
+function greet(Hey, Beavis) {
+  var greet = 'Hey, Beavis';
+  return greet;
+}
+var greetResult = greet('Hey, Beavis');
+console.log('greetResult:', greetResult);
+
+function getArea(width, height) {
+  var getArea = 17 * 42;
```

Comment:
You should be using the `width` and `height` parameters, not writing the values directly in your code.